test/support/flows/maternity_paternity_calculator_flow_test_helper.rb

```diff
@@ -0,0 +1,157 @@
+module MaternityPaternityCalculatorFlowTestHelper
+  def maternity_responses(up_to: nil,
+                          due_date: "2021-05-01",
+                          pay_frequency: "weekly",
+                          pay_per_frequency: 1000,
+                          last_normal_payday: nil,
+                          payday_eight_weeks: nil)
+    due_date = Date.parse(due_date)
+    last_normal_payday = last_normal_payday ? Date.parse(last_normal_payday) : due_date - 4.months
+    payday_eight_weeks = payday_eight_weeks ? Date.parse(payday_eight_weeks) : due_date - 6.months
+
+    responses = { what_type_of_leave?: "maternity",
+                  baby_due_date_maternity?: due_date.to_s,
+                  date_leave_starts?: due_date.to_s,
+                  did_the_employee_work_for_you_between?: "yes",
+                  last_normal_payday?: last_normal_payday.to_s,
+                  payday_eight_weeks?: payday_eight_weeks.to_s }
+
+    responses.merge!(pay_frequency_responses(pay_frequency, pay_per_frequency))
```

Comment:
Hi Kevin, I'm not sure I fully understand the purpose of the suffix arguments on line 48, 87, 118 but I was wondering why we don't seem to have an equivalent for line 19 i.e `suffix: " maternity" `

Reply:
Sure, good question. So the same questions can exist with different names for maternity, paternity and adoption. Those for maternity only exist in an un-suffixed form. For reference:

https://github.com/alphagov/smart-answers/blob/e96abd0d1784866e9ae4f76e5cbdc66e2e311435/app/flows/maternity_paternity_calculator_flow/maternity_calculator_flow.rb#L77

https://github.com/alphagov/smart-answers/blob/e96abd0d1784866e9ae4f76e5cbdc66e2e311435/app/flows/maternity_paternity_calculator_flow/adoption_calculator_flow.rb#L156

https://github.com/alphagov/smart-answers/blob/e96abd0d1784866e9ae4f76e5cbdc66e2e311435/app/flows/maternity_paternity_calculator_flow/paternity_calculator_flow.rb#L250

I assume this is a product of the way the flow was originally written, I'm guessing this was first wrote for maternity and then expanded for paternity and adoption - this probably left some scars along the way, including this inconsistency.